monitor-e2e/test/common.js

```diff
@@ -43,9 +43,21 @@ types.forEach(type => {
         ;({ data } = await axios.get(`${type.baseUrl}/validators`))
       })
 
-      it('home', () => assert(typeof data.home.validators === 'object'))
-      it('foreign', () => assert(typeof data.foreign.validators === 'object'))
-      it('requiredSignaturesMatch', () => assert(data.requiredSignaturesMatch))
+      it('home', () => {
+        assert(typeof data.home.validators === 'object')
+        assert(data.home.validators[validator.address].balance > 0)
+        assert(data.home.validators[validator.address].leftTx > 0)
+        assert(data.home.validators[validator.address].gasPrice > 0)
+      })
+
+      it('foreign', () => {
+        assert(typeof data.foreign.validators === 'object')
+        assert(data.foreign.validators[validator.address].balance > 0)
+        assert(data.foreign.validators[validator.address].leftTx > 0)
+        assert(data.foreign.validators[validator.address].gasPrice > 0)
+      })
+
+      it('requiredSignaturesMatch', () => assert(data.requiredSignaturesMatch, 1))
       it('validatorsMatch', () => assert(data.validatorsMatch))
```

Comment:
Consider to change the number of validators on one side of the bridge by invoking `addValidator` from the validators contract. The monitor should have `false` in `validatorsMatch` when the transaction is included in the block.

Reply:
Right, will do.

Reply:
I have done the changes in similar fashion as #191 

1. The precondition is that `validatorsMatch === true`
2. Then the `addValidator` is executed on only one side of the bridge
3. After `checkAll` and `axios.get`, the expected result is that `validatorsMatch === false`

Reply:
Thanks!